doc/topics/windows/windows-package-manager.rst

```diff
@@ -1221,47 +1194,33 @@ ready for use.
 Troubleshooting
 ***************
 
-My software installs correctly but pkg.installed says it failed
+My software installs correctly but `pkg.installed says it failed
 ===============================================================
 
 If you have a package that seems to install properly, but Salt reports a failure
 then it is likely you have a ``version`` or ``full_name`` mismatch.
 
-Check the exact ``full_name`` and ``version`` as shown in Add/Remove Programs
-(``appwiz.cpl``). Use ``pkg.list_pkgs`` to check that the ``full_name`` and
-``version`` exactly match what is installed. Make sure the software definition
-file has the exact value for ``full_name`` and that the version matches exactly.
-
-Also, make sure the version is wrapped in single quotes in the software
+- Check the  ``full_name`` and ``version`` of the package as shown in Add/Remove Programs
+(``appwiz.cpl``).
+- Use ``pkg.list_pkgs`` to check that the ``full_name`` and
+``version`` exactly match what is installed.
+- Verify that the ``full_name`` and ``version`` of the package in the package definition file
+ matches the full name and version in Add/Remove programs.
+- Ensure that the ``version`` is wrapped in single quotes in the package
 definition file.
 
-Changes to sls files not being picked up
-========================================
+Changes to package definition files not being picked up
+======================================================
```

Comment:
**Nitpick (formatting, blocking):** I think you need an extra = at the end here to make it match the same number of characters as the previous line.

Reply:
Updated